imports/ui/components/study_plan/Tab.jsx

```diff
@@ -189,39 +188,71 @@ export default class Tab extends React.Component {
    */
   renderDropDownMenuSelection(className, iconClassName, iconStyle, text, clickHandler) {
```

Comment:
Multiple parameters are allowed to be passed in, which makes this component very reusable too.